app-network/wpa-supplicant/spec

```diff
@@ -1,5 +1,5 @@
-VER=2.10
-REL=2
+VER=2.11
+REL=1
```

Comment:
Our `REL` starts from 0, for an update like this you can simply remove the line as it defaults to 0.